x-pack/plugins/security_solution/public/resolver/store/data/selectors.test.ts

```diff
@@ -39,29 +40,32 @@ describe('data state', () => {
    */
   const viewAsAString = (dataState: DataState) => {
     return [
-      ['is loading', selectors.isLoading(dataState)],
-      ['has an error', selectors.hasError(dataState)],
+      ['is loading', selectors.isTreeLoading(dataState)],
```

Comment:
These methods were named generically because there was only one resource to fetch at that time. Now that there are multiple, this needs a more specific name. In the future I might try to instead expose a 'resource' object which has methods for determining its state.

Reply:
The test text in this suite changed to match the new selector names.